gateway/test/asserts_test.go

```diff
@@ -103,3 +103,16 @@ func assertValidMutationToken(t *testing.T, token *kv_v1.MutationToken, bucketNa
 func requireValidTimestamp(t *testing.T, ts *timestamppb.Timestamp) {
 	require.NotNil(t, ts)
 }
+
+func assertRpcMessageContains(t *testing.T, err error, msg string) {
```

Comment:
Our API guarantees do not include the specific strings that are returned as part InvalidArguments (they are free to be adjusted whenever we wish).  We've typically intentionally avoided validating these strings in case we want to improve/generic them in the future.  That being said, there might be value in validating that the errors we are producing are sufficiently descriptive to direct developers to how to fix them, thought @Westwooo & @chvck ?

Reply:
I think there is little value if any added by this validation. Since by checking that an appropriate status is returned is implicitly testing that a useful message is returned, based on the assumption that we add an appropriate message when we add the new status.